### ISSUE TITLE
Should not get exception on LSP exit

### ISSUE THREAD

**Issue by sguptatw** [getgauge/gauge-java#196]
**Expected behavior**
Should not get exception on LSP exit

**Actual behavior**
Getting exception
```
SEVERE: Exception while executing runnable io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed@1cb1fa9f
java.lang.NullPointerException
	at com.thoughtworks.gauge.LspServer.killProcess(LspServer.java:142)
	at gauge.messages.lspServiceGrpc$MethodHandlers.invoke(lspServiceGrpc.java:825)
	at io.grpc.stub.ServerCalls$UnaryServerCallHandler$UnaryServerCallListener.onHalfClose(ServerCalls.java:171)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.halfClosed(ServerCallImpl.java:283)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed.runInContext(ServerImpl.java:707)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1135)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:844)
```

**Steps to replicate**
* Create a gauge-java project
`gauge init java`
* Open in VSCode IDE
* Reload the project
* Open the lsp logs
> The above error can be observed

Blocks https://github.com/getgauge/gauge-vscode/issues/83

**Comment by sguptatw** [comment]
[Blocker] Latest nightly available is 0.7.0.nightly-2018-12-04

**Comment by sguptatw** [comment]
Working as expected on version
```
0.7.0.nightly-2018-12-10
```
